### DISCUSSION TITLE
Allow to hide folder button on dashboard

### DISCUSSION THREAD

**Discussion by absgrlyp** [grafana/grafana#38241] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
Please add an option to hide folder button on Dashboard. This is for sure good for many dashboards, but with such a change there should be option to revert it.

**Why is this needed**:
The interactive dashboard with jumps between them don't need such button and user can be confused when see a list of dashboards, especially when some of them might not work properly without parameters passed.

Grafana 7.5.0 beta1
![image](https://user-images.githubusercontent.com/29247982/111512010-8fd1f580-874f-11eb-823e-455eebc4b0ac.png)

Grafana 7.3.2
![image](https://user-images.githubusercontent.com/29247982/111512046-9d877b00-874f-11eb-98ac-7febb4611019.png)

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
